File: main/创建&重置settings.ini配置文件.c
#include <stdio.h>

int main() {
    FILE *configfile = fopen("settings.ini", "w");
    if (configfile == NULL) {
        printf("Error opening file for writing.");
        return 1;
    }

    fprintf(configfile, "[Settings]\n");
    fprintf(configfile, "middle_font_size = 512\n");
    fprintf(configfile, "middle_font_color = 255,255,255,255\n");
    fprintf(configfile, "text_position_x = 0\n");
    fprintf(configfile, "text_position_y = 0\n");
    fprintf(configfile, "bottom_font_size = 19\n");
    fprintf(configfile, "background_color = 0,0,0,255\n");

    fclose(configfile);

    return 0;
}

File: main/创建&重置settings.ini配置文件.cs
using System;
using System.IO;

class Program
{
    static void Main()
    {
        using (StreamWriter configfile = new StreamWriter("settings.ini"))
        {
            if (configfile == null)
            {
                Console.WriteLine("Error opening file for writing.");
                return;
            }

            configfile.WriteLine("[Settings]");
            configfile.WriteLine("middle_font_size = 512");
            configfile.WriteLine("middle_font_color = 255,255,255,255");
            configfile.WriteLine("text_position_x = 0");
            configfile.WriteLine("text_position_y = 0");
            configfile.WriteLine("bottom_font_size = 28");
            configfile.WriteLine("background_color = 0,0,0,255");
        }

    }
}